src/main/java/com/synopsys/integration/detect/RunBeanConfiguration.java

```diff
@@ -139,7 +141,7 @@ public AirGapPathFinder airGapPathFinder() {
 
     @Bean
     public CodeLocationNameGenerator codeLocationNameService() {
-        final String codeLocationNameOverride = detectConfiguration.getValueOrEmpty(DetectProperties.Companion.getDETECT_CODE_LOCATION_NAME()).orElse(null);
```

Comment:
Not in love with removing all the finals, but since we don't have an official rule on it, it's fine.

Reply:
My save actions are configured based on the _Engineer Onboarding_ on Confluence, but if you guys have a different preference in Detect, I can update the settings for the project accordingly. I assumed the finals were just left over from when we still had that convention mandated.

Reply:
If you could add them back for now that would be great.